cleanlab/internal/validation.py

```diff
@@ -206,3 +207,53 @@ def labels_to_array(y: Union[LabelLike, np.generic]) -> np.ndarray:
             raise ValueError(
                 "List of labels must be convertable to 1D numpy array via: np.ndarray(labels)."
             )
+
+
+def assert_valid_inputs_multiannotator(
+    labels_multiannotator: Union[pd.DataFrame, np.ndarray], pred_probs: np.ndarray
```

Comment:
Whoops, I missed the first check `pred_probs is not None`:

```suggestion
    labels_multiannotator: Union[pd.DataFrame, np.ndarray], pred_probs: Optional[np.ndarray]
```